ChattoApp/ChattoApp/Source/Text Messages/DemoTextMessageViewModel.swift

```diff
@@ -44,6 +44,9 @@ public class DemoTextMessageViewModelBuilder: ViewModelBuilderProtocol {
     public func createViewModel(textMessage: DemoTextMessageModel) -> DemoTextMessageViewModel {
         let messageViewModel = self.messageViewModelBuilder.createMessageViewModel(textMessage)
         let textMessageViewModel = DemoTextMessageViewModel(textMessage: textMessage, messageViewModel: messageViewModel)
+        // Best place to decide whether user having avart might be in the presenter. Here just for demo purpose
+        // Because we might only want to display avatar when showTail in the nessage bubble
+        textMessageViewModel.avatarImage = UIImage(named: "userAvatar")
```

Comment:
Mmm... Haven't thought of this. You're right, trying to implement some kind of logic in which the avatar is not shown in every cell looks complicated as it is now. I'd rather add a new attribute ChatItemDecorationAttributes.canShowAvatar and replicate what it's done with the tail